src/lib/koa-websocket.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
```

Comment:
Gotcha. In that case, sounds fine to copy the code locally and remove the dependency on that package